src/shared_modules/content_manager/src/components/CtiApiDownloader.hpp

```diff
@@ -193,6 +190,9 @@ class CtiApiDownloader final : public AbstractHandler<std::shared_ptr<UpdaterCon
                 throw std::runtime_error {exceptionMessage};
             }};
 
+        const auto fullUrl {m_url + queryParameters};
+        logDebug2(WM_CONTENTUPDATER, "Attempting to download content from: '%s'", fullUrl.c_str());
```

Comment:
How about 
```suggestion
        logDebug2(WM_CONTENTUPDATER, "Downloading content from: '%s'", fullUrl.c_str());
```
Current sample log:
```
2023/11/30 18:22:09 wazuh-modulesd:content-updater[1069170] CtiApiDownloader.hpp:192 at performQueryWithRetry(): DEBUG: Attempting to download content from: 'https://cti.wazuh.com/api/v1/catalog/contexts/vd_1.0.0/consumers/vd_4.8.0/changes?from_offset=214000&to_offset=215000'
2023/11/30 18:22:24 wazuh-modulesd:content-updater[1069170] CtiApiDownloader.hpp:192 at performQueryWithRetry(): DEBUG: Attempting to download content from: 'https://cti.wazuh.com/api/v1/catalog/contexts/vd_1.0.0/consumers/vd_4.8.0/changes?from_offset=215000&to_offset=216000'
2023/11/30 18:22:37 wazuh-modulesd:content-updater[1069170] CtiApiDownloader.hpp:192 at performQueryWithRetry(): DEBUG: Attempting to download content from: 'https://cti.wazuh.com/api/v1/catalog/contexts/vd_1.0.0/consumers/vd_4.8.0/changes?from_offset=216000&to_offset=217000'
```

I think we can just say that it is `downloading`, if there is an error it will be logged.
What do you think?